index.vue

```diff
@@ -53,7 +49,8 @@
   import ScbdFooter from './src/components/cbd-nav/footer.vue'
   import CopCountDown from './src/components/countdown/cop-count-down.vue';
   import dateSelector from  './src/components/inputs/dateSelector.vue'
-  import { ref, provide } from 'vue';
+  import kmnotes from  './src/components/inputs/notes.vue'
```

Comment:
shoudl use camel casing
```suggestion
  import kmNotes from  './src/components/inputs/notes.vue'
```

Reply:
done